distributed/security.py

```diff
@@ -49,7 +49,7 @@ class Security(object):
     __slots__ = tuple(_fields)
 
     def __init__(self, **kwargs):
-        self._init_from_dict(config)
+        self._init_from_dict(dask.config.config)
```

Comment:
Did you mean to double `.config` here?

Reply:
Yes, the dask.config module has a config dict.  See https://github.com/dask/dask/pull/3432 for the co-PR